pkgs/applications/graphics/ktikz/default.nix

```diff
@@ -0,0 +1,74 @@
+{ withKDE ? true,
```

Comment:
is this parameter necessary? I suppose checking whether kdelibs is null or not would suffice?
Anyway, I would like to see this package in.